changelogs/fragments/137-api.yml

```
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - api_info, api_modify - support API paths - ``interface ethernet poe``, ``interface gre6``, ``interface vrrp`` and also support for all preivously missing fileds of entries in ``ip dhcp-server``
+bugfixes:
+  - api_info, api_modify - ``address-pool`` field of entries in API path ``ip dhcp-server`` is not required anymore
```

The required should only affect api_modify, api_info shouldn't care about it:
```suggestion
  - api_modify - ``address-pool`` field of entries in API path ``ip dhcp-server`` is not required anymore (https://github.com/ansible-collections/community.routeros/pull/137).
```